src/Parser/Hunter/Survival/Modules/Features/CooldownThroughputTracker.js

```diff
@@ -0,0 +1,31 @@
+import SPELLS from 'common/SPELLS/index';
+
+import CoreCooldownThroughputTracker, { BUILT_IN_SUMMARY_TYPES } from 'Parser/Core/Modules/CooldownThroughputTracker';
+
+class CooldownThroughputTracker extends CoreCooldownThroughputTracker {
+  static cooldownSpells = [
+    ...CooldownThroughputTracker.cooldownSpells,
+    {
+      spell: SPELLS.MONGOOSE_FURY,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+    {
+      spell: SPELLS.ASPECT_OF_THE_EAGLE,
+      summary: [
+        BUILT_IN_SUMMARY_TYPES.DAMAGE,
+      ],
+    },
+  ];
+
+  static ignoredSpells = [
+    ...CooldownThroughputTracker.ignoredSpells,
+    SPELLS.GOLGANNETHS_VITALITY_RAVAGING_STORM.id,
+    SPELLS.GOLGANNETHS_VITALITY_THUNDEROUS_WRATH.id,
```

Comment:
Golganneth's stuff should probably be in the core ignored spells instead of here

Reply:
Fixed this - and fixed the merge conflict that came up after #1210